content/blog/2024-09-16-valkey-8-ga.md

```diff
@@ -0,0 +1,26 @@
++++
+title=  "Generally Available: Valkey 8.0.0"
+description = "Today marks a milestone for the Valkey project: the first major release."
+date= 2024-09-16 01:01:01
+authors= ["kyledvs"]
++++
+
+The first ever release of Valkey, 7.2.5, became generally available more than 5 months ago.
+While the initial release was a milestone, it focused on compatibility and license continuity; bringing no new features to the table.
+Today marks a different milestone for the Valkey project: the first major release.
+Valkey 8.0.0 continues the the traditions of the seven major versions of Redis that precede it by bringing improvements to speed and efficiency alongside new features.
```

Comment:
```suggestion
Valkey 8.0.0 continues the traditions of the seven major versions of Redis that precede it by bringing improvements to speed and efficiency alongside new features.
```

Reply:
Thanks. I've read this thing 20 times and I didn't didn't see see this this.